README.md

```diff
@@ -19,6 +19,11 @@ If you are already using a linter to identify coding standards problems in your
 code, you know that fixing them by hand is tedious, especially on large
 projects. This tool does not only detect them, but also fixes them for you.
 
+## Supported versions
```

Comment:
I wonder if we can put extra detail here about not-yet-supported versions, that would prevent the new tickets to be created?

Reply:
I like it... Any ideas? How about to add a note below the PHP versions list, something like this:

> Each new PHP version requires a huge effort to support the new syntax. That's why the latest PHP version might not be supported yet. If you need it, please, consider supporting the project in any convenient way: [GitHub sponsorship](https://github.com/sponsors/keradus) or with code contribution.

Reply:
I like it, but I will prioritize support that way: `in any convenient way, for example with code contribution or reviewing existing RPs`